### DISCUSSION TITLE
Welcome to Nixpacks Discussions!

### DISCUSSION THREAD

**Discussion by coffee-cup** [railwayapp/nixpacks#119] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community and discuss various things about Nixpacks. Feel free to create discussion for any questions or proposals you have.

**Comment by aleksrutins** [comment]
Suggestion: Should there be a separate "RFCs" category for RFCs? Just to unclutter things a bit.

**Reply by coffee-cup** [reply]
Ya good idea
